addons/mail/models/mail_message.py

```diff
@@ -912,12 +913,13 @@ def _message_format(self, fnames, format_reply=True, legacy=False):
                 'linkPreviews': message_sudo.link_preview_ids._link_preview_format(),
                 'messageReactionGroups': reaction_groups,
                 'record_name': record_name,
+                'pinned_at': message_sudo.pinned_at,
             })
 
         return vals_list
 
     @api.model
-    def _message_fetch(self, domain, max_id=None, min_id=None, limit=30):
+    def _message_fetch(self, domain, max_id=None, min_id=None, limit=30, order=None):
```

Comment:
Would be nice to reword `max` and `min` to `before` and `after`, so it's easier to understand the expected ranges of messages to be fetched when calling this method.

Reply:
I agree this would be more intelligible, @seb-odoo do you agree to change the current API?